task/harden-concourse-tasks/scripts/build.sh

```diff
@@ -36,7 +36,7 @@ echo "Updating system package registry"
 add-apt-repository ppa:rmescandon/yq
 apt-get -y update
 
-echo "Installing basic libraries and development utilities"
+# echo "Installing basic libraries and development utilities"
```

Comment:
Let's either drop this entirely or uncomment it? We can always grab it from git history if we want it